numpyro/distributions/continuous.py

```diff
@@ -3068,3 +3071,128 @@ def entropy(self) -> ArrayLike:
         return jnp.broadcast_to(
             0.5 + 1.5 * jnp.euler_gamma + 0.5 * jnp.log(16 * jnp.pi), self.batch_shape
         ) + jnp.log(self.scale)
+
+
+class CirculantNormal(TransformedDistribution):
+    """
+    Multivariate normal distribution with covariance matrix that is positive-definite
+    and circulant [1], i.e., has periodic boundary conditions. Circulant matrices can be
```

Comment:
I think I understand what is a circulant matrix (from the wikipedia page) but I don't understand why this is a MVN distribution (it seems to be a transformed normal distribution based on the implementation) and why this matrix is called covariance matrix, and what "positive-definite" means here. Could you clarify?

It would be helpful to add formula in the docstrings like https://num.pyro.ai/en/stable/distributions.html#numpyro.distributions.continuous.Levy